exercise/submission_models.py

```diff
@@ -829,6 +829,22 @@ def is_grader_stable(self):
         return not (total_retries and total_retries > settings.GRADER_STABLE_THRESHOLD)
 
 
+    def get_exercise_names_if_grader_is_unstable(self, instance):
+        total_retries_per_exercise = self.values(
+            'submission__exercise__name',
```

Comment:
If multiple exercises in the course instance have identical names (for example, some courses use the name "feedback" a lot), then does this query separate those exercises or does it combine them into one due to the identical name?

Reply:
The query combines exercises with identical names under one name along with their combined number of retries.

Reply:
I was thinking that it should separate each exercise, but maybe in this context it does not really matter. After all, the function is only counting a sum from the exercises and listing the most problematic exercises (without the more descriptive identifiers like hierarchical names). So this could be just fine like this.